### DISCUSSION TITLE
Unable to make allow resize work

### DISCUSSION THREAD

**Discussion by parikhharsh96** [julianpoemp/ngx-jodit#33] [Q&A] [answered]
Hello, 
First of all I would like to thank you for your amazing work. 
I am going to ask a very silly question 
I am using v4 with ng17

These are the options in the .ts file

 options : JoditConfig = {
    allowResizeX: true,
  allowResizeY: false
  };

This is the html 
<ngx-jodit class="editor" [(value)]="value" [options]="options" #ngxJodit></ngx-jodit>

But I am unable to get the resizing ability added. 
Can you please help?

**Comment by julianpoemp** [comment]
Hello @parikhharsh96,

did you import the plugin "resize-handler"? For more information see

https://github.com/julianpoemp/ngx-jodit/blob/main/libs/ngx-jodit/README.md#how-to-import-plugins

In your case you should place the following import statement:

````typescript
import 'jodit/esm/plugins/resize-handler/resize-handler.js';
````

**Reply by parikhharsh96** [reply]
Hi @julianpoemp,
I added the import but still it is not working. Could you please help me if I need to make any changes to the options after adding the import?

**Reply by julianpoemp** [reply]
@parikhharsh96 according to Jodit docs you need to set the height attribute in options. For example:

````typescript
{
  height: 300,
  allowResizeX: true,
  allowResizeY: false
}
````

**Reply by julianpoemp** [reply]
> @parikhharsh96 according to Jodit docs you need to set the height attribute in options. For example:
> 
> ```ts
> {
>   height: 300,
>   allowResizeX: true,
>   allowResizeY: false
> }
> ```

did you get it working?

**Reply by parikhharsh96** [reply]
No this didn't help  :(

**Reply by julianpoemp** [reply]
I tested it and it's working for me. I can change the size along the x axis dragging the bottom right corner of the editor.

Are you sure you imported the plugin and set height in options like I said before? 🤔

**Reply by parikhharsh96** [reply]
Maybe I might not have imported it correctly. Can you please share the correct snippet for import as well?

**Reply by julianpoemp** [reply, marked as answer]
see working example here. I added both suggestions I said before:
https://stackblitz.com/edit/stackblitz-starters-eu7k8e?file=src%2Fmain.ts

**Reply by parikhharsh96** [reply]
I was importing it as - 
import resizeHandler from 'jodit/esm/plugins/resize-handler/resize-handler.js'
but adding your import statement worked
Thank you so much!
